composition/http_content_loader.go

```diff
@@ -23,13 +23,14 @@ func NewHttpContentLoader() *HttpContentLoader {
 }
 
 // TODO: Should we filter the headers, which we forward here, or is it correct to copy all of them?
-func (loader *HttpContentLoader) Load(fd *FetchDefinition) (Content, error) {
+func (loader *HttpContentLoader) Load(fd *FetchDefinition) (Content, error, int) {
```

Comment:
Same as above:
Since the Http Status Code is already part of Content.httpStatusCode,
there is no need to double it here in the method return parameters.